.github/workflows/testsuite.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Run Tests
         run: |
           pip install .[test]
+          conda list
```

Comment:
The actual test output can be quite verbose. I wonder if the install and conda list should be a separate step?

Reply:
Good point. I've moved it to its own step.